### PR TITLE
cleaned up a lot of code inside generativeai-cleanup based on comments

### PR DESCRIPTION
**Pull request by erickang21**

### CODE REVIEW

File: components/practice/true-false.js [hunk 1]
```diff
@@ -375,7 +375,7 @@ class TrueFalsePractice extends RequesterMixin(Localizer(LitElement)) {
 			}
 			const contentFormatted = content.trim().replace('"""', '');
 			if (fieldToFill === fields.question) {
-				this.activityData.questions[0].text = contentFormatted;
+				//this.activityData.questions[0].text = contentFormatted;
```
Comment (JamesVaChao) [inline components/practice/true-false.js:378]: don't comment this, just delete it. Also do a full testing of it. Make sure you save the question and try to edit it to verify that its working/saving correctly.

Comment (erickang21) [reply components/practice/true-false.js:378]: did a full test, including inserting it, and i can confirm it works.

File: components/plugins/practice.js [hunk 2]
```diff
@@ -449,7 +449,7 @@ class PracticeDialog extends RequesterMixin(Localizer(LitElement)) {
 		}
 
 		.d2l-config {
-			display: flex;
+
```
Comment (ianmclean2011) [inline components/plugins/practice.js:452]: Extra space here can be removed.

Comment (ianmclean2011) [reply components/plugins/practice.js:452]: Also, this change appears to impact pre-existing styling, so just want to make sure it isn't breaking anything Practices related?

File: components/practice/dropdown-blanks.js [hunk 3]
```diff
@@ -672,24 +672,20 @@ class DropdownBlanksPractice extends RequesterMixin(Localizer(LitElement)) {
...
-				this.activityData.questions[0].text = contentFormatted;
 				this._updateValue('questions:0:text', contentFormatted);
 				this.questionText = contentFormatted;
 
 			} else if (fieldToFill === fields.answer) {
 				const answerIndex = this.activityData.questions[0].answers.map(function(answer) { return answer.id; }).indexOf(answerId);
 				if (answerIndex > -1) {
-					this.activityData.questions[0].answers[answerIndex].answers[0].text = contentFormatted;
 					this._updateValue(`questions:0:answers:${answerIndex}:answers:0:text`, contentFormatted);
-					this._updateActivity();
```
Comment (ianmclean2011) [inline components/practice/dropdown-blanks.js:684]: Curious ... is there a reason this call to `_updateActivity()` is being removed, but other calls in other areas of code aren't being removed?

File: components/plugins/practice.js (hunk 2 above)
Comment (erickang21) [reply components/plugins/practice.js:452]: seems like this should be kept based on my testing. i'll fix this

File: components/practice/dropdown-blanks.js (hunk 3 above)
Comment (erickang21) [reply components/practice/dropdown-blanks.js:684]: on second thought, this isn't necessary for cleanup so i will add it back

File: components/plugins/practice.js [hunk 4]
```diff
@@ -1012,7 +1012,7 @@ class PracticeDialog extends RequesterMixin(Localizer(LitElement)) {
...
 
 			case 3:
-				return html`<d2l-true-false-practice id="d2l-true-false-practice" @updateValue="${this._updateValue}" @activityUpdate="${this._activityUpdated}" @activitySave="${this._saveActivity}" @togglePreview="${this._togglePreview}" @handleClose="${this._checkForChangesBeforeClose}" .activityData="${this.activityData}" ?display-feedback="${this._modifiers[0].value}" ?has-hint="${this._modifiers[1].value}" ?use-enumeration="${this._modifiers[2].value}" .editorText="${editorText}" .titleText="${titleText}" .summary="${this.summary}"></d2l-true-false-practice>`;
+				return html`<d2l-true-false-practice id="d2l-true-false-practice" @activityUpdate="${this._activityUpdated}" @activitySave="${this._saveActivity}" @togglePreview="${this._togglePreview}" @handleClose="${this._checkForChangesBeforeClose}" .activityData="${this.activityData}" ?display-feedback="${this._modifiers[0].value}" ?has-hint="${this._modifiers[1].value}" ?use-enumeration="${this._modifiers[2].value}" ?display-short-description="${this._modifiers[3].value}" .editorText="${editorText}" .titleText="${titleText}" .summary="${this.summary}"></d2l-true-false-practice>`;
```
Comment (ianmclean2011) [inline components/plugins/practice.js:1015]: I believe @JamesVaChao just removed the display-short-description part.

**Review by ianmclean2011 — APPROVED** [review]
Looks ok to me.

File: components/practice/dropdown-blanks.js (hunk 3 above)
Comment (JamesVaChao) [reply components/practice/dropdown-blanks.js:681]: So are we removing this or adding it back? and jsut to confirmed you tested it a few times to make sure things are updating properly?

Comment (JamesVaChao) [reply components/practice/dropdown-blanks.js:681]: you can either keep it (and test it) or remove these change and not have to test that part

**Pull request merged**